packaging/common/KNI/README.md.in

```diff
@@ -0,0 +1,100 @@
+
+# Khiops Native Interface examples @PROJECT_VERSION@
+
+The purpose of Khiops Native Interface (KNI) is to allow a deep integration of Khiops in information systems, by the mean of the C programming language, using a dynamic link library (DLL). This relates especially to the problem of model deployment, which otherwise requires the use of input and output data files when using directly the Khiops tool in batch mode. See the [Khiops Guide](https://khiops.com/pdf/KhiopsGuide.pdf) for an introduction to dictionary files, dictionaries, database files and deployment.
+
+The Khiops deployment features are thus made public through an API with a DLL. Therefore, a Khiops model can be deployed directly from any programming language, such as C, C++, Java, Python, Matlab... This enables model deployment in real time application (e.g. scoring in a marketing context, targeted advertising on the web) without the overhead of temporary data files or launching executables.
```

Comment:
Matlab... -> Matlab, etc.
This enables model deployment in real time application -> This enable real time model deployment.

The parenthese with examples I would transformit in to a phrase after launching excutables:
- This is critical for certain applications in Marketing or Targeted Advertising on the web.